src/app/modules/browse/browse.tpl.html

```diff
@@ -70,7 +70,7 @@
                                 <div>{{bookMessage}}</div>
                                 <div ng-show="numHiddenBooks || bookCount!=0">
                                     <div class="space-after-book-message"></div>
-				    <div ng-show="numHiddenBooks">{{hiddenBooksMessage}}  <a href="" ng-click="toggleAllLicenses()">{{hiddenBooksToggleMessage}}</a></div>
+				    <div ng-show="numHiddenBooks"><span>{{hiddenBooksMessage}}</span>  <a href="" ng-click="toggleAllLicenses()"><span>{{hiddenBooksToggleMessage}}</span></a></div>
```

Comment:
I'm curious, is the second span required? Seems that it was already content of an element.  

On a different issue, I wonder how long it is before someone removes  these spans that have no ids, classes, etc. because they don't "appear" to do anything.